nautobot_data_validation_engine/__init__.py

```diff
@@ -23,6 +23,7 @@ class NautobotDataValidationEngineConfig(PluginConfig):
     required_settings = []
     default_settings = {}
     caching_config = {}
+    min_version = "1.2"
```

Comment:
Would be good to also add `nautobot ~1.2` in `pyproject.toml` as well (currently missing), if that doesn't open too much of a can of worms to do so at this time...